src/test_lib_json/main.cpp

```diff
@@ -1,9 +1,10 @@
-// Copyright 2007-2010 Baptiste Lepilleur
```

Comment:
This has been fixed. (idk how github code review works, I guess I have to respond to mark this as done, since it still has the orange blob saying work needs to be done?)